covalent/_dispatcher_plugins/local.py

```diff
@@ -122,6 +122,11 @@ def wrapper(*args, **kwargs) -> str:
             # To access the disable_run passed to the dispatch function
             nonlocal disable_run
 
+            if not isinstance(orig_lattice, Lattice):
+                message = f"Dispatcher expected a Lattice, received {type(orig_lattice)} instead."
+                app_log.error(message)
+                raise ValueError(message)
```

Comment:
I think a `TypeError` would be better here.

Reply:
Aah, sorry, my bad, `TypeError` it should be.